source/extensions/filters/http/common/ratelimit_headers.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "envoy/http/header_map.h"
+
+#include "source/common/singleton/const_singleton.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Common {
+namespace RateLimit {
```

Comment:
nit: newline after namespace (would have thought clang-tidy would complain here).

Reply:
Thanks. I will run clang-tidy and fix related issues.